Include pull request reviews in database

MaineC opened the pull request:
Problem description: When commenting on pull requests, Github offers the option to bundle several comments together into a pull request review. As far as I can see, those reviews don't make it into the database so far. This PR adds the results of the https://octokit.github.io/octokit.rb/Octokit/Client/PullRequests.html#pull_requests_comments-instance_method call.

See https://github.com/MaineC/sandbox/pull/1 for an example some of the various types of comments that can be made on a pull request.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

hyandell commented on refresh-dashboard.rb:
```diff
@@ -29,8 +29,8 @@
     Dir.glob(xmlFiles).each do |inputFile|
-      stylesheet = LibXSLT::XSLT::Stylesheet.new( LibXML::XML::Document.file(xsltFile))
       xml_doc = LibXML::XML::Document.file(inputFile)
+      stylesheet = LibXSLT::XSLT::Stylesheet.new( LibXML::XML::Document.file(xsltFile))
```
Any reason for this change? i.e. a nefarious bug?

MaineC replied:
No. Only me forgetting to revert things I changed while debugging a weird combination of Ubuntu/ xslt/ ruby:

https://github.com/xml4r/libxslt-ruby/pull/13

https://github.com/tonyfg/libxslt-ruby/pull/1

hyandell commented on github-sync/sync.rb:
```diff
@@ -69,6 +69,10 @@ def github_sync(context, run_one)
     # TODO: Move to being based on the async code
     sync_issue_comments(context, sync_db)
   end
+  if(not(run_one) or run_one=='github-sync/pr-reviews')
```
Mental note that I'll also need to implement the async side of this.

MaineC replied:
Happy to help here - it will take me a while to figure things out though: My Ruby experience is pretty much non-existent ;)

hyandell commented:
Thank you Isabel - sorry for the delay :) 

Just the one question added, and I need to test this out, including seeing what happens when it's run with --xsync.

MaineC commented:
> Thank you Isabel - sorry for the delay :)

No need for sorry - I know what busy schedules look like.


> Just the one question added, and I need to test this out, including seeing what happens when it's run 
> with --xsync.

Ah - so xsync is what triggers the -tng code path to be executed :)

MaineC commented:
I have simplified the implementation quite a bit (and discovered a bug in the earlier implementation with timestamp handling in my original patch). I also added support for synching review comments also when working with --xsync.

If you want to quickly check things, feel free to use `https://github.com/MaineCSandboxOrg/Sandbox` which has three comments on a pull request:

1. A regular comment.
2. A comment that is attached to a PR review and was added before hitting Submit.
3. A comment that was added to a PR as part of a review.

Before the patch above, only 1. will be stored. After the patch, 1. and 2. will be stored.

hyandell commented:
Thanks for waiting Isabel - the latest code worked great for me. Using this https://github.com/amazon-archives/serverless-image-resizing/pull/65 I went from 5 saved comments to 11, and saw no errors.
